docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,284 @@
+
+### Decide on the structure of your folders and certificates
+
+Whether you are running in a test or production environment, it is critically important that you maintain a consistent and coherent structure for managing your folders and certificates. While it is not a strict Fabric necessity to use the same patterns everywhere (as long as your paths are correct whenever you reference them, for example when bootstrapping a node, Fabric can consume them), certificate pathing errors are among the most common errors faced by Fabric users. Forethought and consistency can dramatically reduce these issues.
+
+Note that production deployments might include structures we won't show here. For example, it might include a folder for **gateways**, allowing an admin to easily see which organizations and nodes and clients are associated with each network. Similarly, you might see a **smart contracts** folder containing the smart contracts associated with a network.
+
+The method for organizing your folders and certificates we describe here is not mandatory, but you will find it helpful, as it is consistent with the rest of this topic as well as with the [CA deployment guide](ca-deploy.html). Most importantly, it organizes your structures around the **organization** that owns and manages them. While it might be natural to think of your deployments as being organized around physical structures like peers and ordering nodes, it is actually the organization that is the centralizing figure. Especially since not all network participants will necessarily own a node.
+
+The structures and methods presented here also represent best practices, preventing cases where, for example, Fabric expects an MSP folder to be called `msp` and, if a different name is used, the name of the folder will have to be changed in the relevant YAML file.
+
+#### Folder structure for operating the Fabric CA client
+
+While a consistent structure is important for the certificates you will get back from a CA (which you will use when creating nodes and acting as an admin), it is also important CA admins who will be using a single Fabric CA client to connect to multiple CAs as an admin. This is because unlike organization admins, in which a single identity can be used as the admin of as many nodes as you need, each CA necessarily has a separate admin that is registered when it is bootstrapped and later enrolled.
+
+This is why if you are connecting to different CA servers as an admin from the same CA client, when you use the `--mspdir` flag, you also **must** include the `-u` flag to target the correct CA server. This will allow you to specify the correct CA admin credentials for the CA you are connecting to.
```

Comment:
This paragraph is pretty much the same as the `Note` in line 25. Since we talk about this here, not sure if the Note is necessary. But, I am okay either way.